Adding some instructions for IntelliJ and Maven (#18)
gurkenlabs/litiengine-docs: getting-started/build-systems.md

* two files are required in a Gradle project's root folder: `build.gradle`, containing the build tasks, and `settings.gradle`, containing Gradle settings.

### Example: IntelliJ + Gradle
If you chose [Intellij](https://www.jetbrains.com/idea/) and [Gradle](https://gradle.org/), follow these steps:

> > If you're on linux or mac, I would hightly recommend to use [SKDMAN!](https://sdkman.io/). If you installed SDKMAN!, type `sdk install gradle` in your terminal. The latest recommend version of Gradle will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage). **If you use SDKMAN! skip the next three steps**.

Reviewer: ```suggestion
> If you're on linux or mac, you can also use [SDKMAN!](https://sdkman.io/) to manage your Gradle installation. If you installed SDKMAN!, type `sdk install gradle` in your terminal. The latest recommend version of Gradle will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage). **If you use SDKMAN! skip the next three steps**.
```

> * [download Gradle](https://gradle.org/install/).

Author: IntelliJ already uses a wrapper automatically so the installation of gradle (or maven) is obsolete. I added information about that in my new pr

* create a new directory `C:\Gradle` and unzip the downloaded file there. 
* add `C:\Gradle\[your-gradle-version]\bin` to your system's *PATH* environment variable.
* usually the Gradle plugin is already installed in IntelliJ, if not, search for it in the plugin marketplace.
* create a new project and select `Gradle` under `Build system`.
* two files are required in a Gradle project's root folder: `build.gradle`, containing the build tasks, and `settings.gradle`, containing Gradle settings.

![intellij_gradle](/getting-started/img/intellij_gradle.png)

### Example: Intellij + Maven
If you want to use [Intellij](https://www.jetbrains.com/idea/) and [Maven](https://maven.apache.org/) follow these steps:

> > If you're on linux or mac, I would hightly recommend to use [SKDMAN!](https://sdkman.io/). If you installed SDKMAN!, type `sdk install maven` in your terminal. The latest recommend version of Maven will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage). **If you use SDKMAN! skip the next three steps**.

Reviewer: ```suggestion
> If you're on linux or mac, you can also use [SDKMAN!](https://sdkman.io/) to manage your Maven installation. If you installed SDKMAN!, type `sdk install maven` in your terminal. The latest recommend version of Maven will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage). **If you use SDKMAN! skip the next three steps**.
```

### Example: IntelliJ + Gradle
To use [Intellij](https://www.jetbrains.com/idea/) and [Gradle](https://gradle.org/), follow these steps:

> > If you're on linux or mac, you can also use [SKDMAN!](https://sdkman.io/) to manage your Gradle installation. As soon you installed SDKMAN!, type `sdk install gradle` in your terminal. The latest recommend version of Gradle will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage). **In case you use SDKMAN! skip the next three steps**.

Reviewer: ```suggestion
> If you're on linux or mac, you can also use [SKDMAN!](https://sdkman.io/) to manage your Gradle installation. Once you installed SDKMAN!, type `sdk install gradle` in your terminal. The latest recommend version of Gradle will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage). **In case you use SDKMAN! skip the next three steps**.
```

### Example: Intellij + Maven
Getting ready with [Intellij](https://www.jetbrains.com/idea/) and [Maven](https://maven.apache.org/):

> > If you're on linux or mac, you can also use [SKDMAN!](https://sdkman.io/) to manage your Maven installation. As soon you installed SDKMAN!, type `sdk install maven` in your terminal. The latest recommend version of Maven will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage). **In case you use SDKMAN! skip the next three steps**.

Reviewer: ```suggestion
> If you're on linux or mac, you can also use [SKDMAN!](https://sdkman.io/) to manage your Maven installation. Once you've installed SDKMAN!, type `sdk install maven` in your terminal. The latest recommend version of Maven will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage). **In case you use SDKMAN! skip the next three steps**.
```
